### PR TITLE
Move selected editors instead of only activ ones

### PR DESCRIPTION
**Pull request by benibenj**
Refactor the editor movement logic to support moving multiple selected editors instead of just the active editor. 

related: #236269

@bpasero I did not change the command ID as extensions might be using them. The args and description still mention active editor and not selected editors. Should we keep it that way or mention selected editors?

### CODE REVIEW

File: src/vs/workbench/browser/parts/editor/editorCommands.ts [hunk 1]
```diff
@@ -178,42 +178,55 @@ function registerActiveEditorMoveCopyCommand(): void {
...
-		const activeEditorPane = accessor.get(IEditorService).activeEditorPane;
-		if (activeEditorPane) {
+		const activeGroup = accessor.get(IEditorGroupsService).activeGroup;
+		const selectedEditors = activeGroup.selectedEditors;
+		if (selectedEditors.length > 0) {
 			switch (args.by) {
 				case 'tab':
 					if (isMove) {
-						return moveActiveTab(args, activeEditorPane);
+						moveTabs(args, activeGroup, selectedEditors);
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/editor/editorCommands.ts:206]: We used to have a `return` here?

**Review by bpasero — CHANGES_REQUESTED** [review]
@benibenj yes, command Ids should not change, but for extensions this command is still doing as expected, no? Just it considers all selected tabs now?

File: src/vs/workbench/browser/parts/editor/editorCommands.ts (hunk 1 above)
Comment (benibenj) [reply src/vs/workbench/browser/parts/editor/editorCommands.ts:206]: Sure, can bring it back

**Comment by benibenj** [comment]
Yes, behaviour is the same as before. With the only change that if there are selected editors, the move operation will also apply to them

**Review by bpasero — APPROVED** [review]

**Pull request merged**